hangman.rb

```diff
@@ -0,0 +1,11 @@
+def hangman(word, guess_array)
+  result = word.split('').map do |el|
+    guess_array.include?(el) ? el : "_"
```

Comment:
Strings have an iterative method that allow you to loop by each character called...`each_char`, so that you don't have to create a new array, and then map it.

You could then simply build result as a string from the beginning, avoiding the `join` call, too.

```ruby
  result = ""
  word.each_char do |char|
    guess_array.include?(char) ? result += char : result += "_"
  end
  puts result
```